tonicpow/admin/class-tonicpow-admin.php

```diff
@@ -37,255 +37,289 @@ class Tonicpow_Admin
 					<h4 style="margin: 0;">' . $campaign->{'title'} . '</h4>' .
-						$campaign->{'description'} . '
+					     $campaign->{'description'} . '
 					<br />';
 
-					foreach ($campaign->{'goals'} as $goal) {
+					foreach ( $campaign->{'goals'} as $goal ) {
 						// Add the goal to the goal selector
-						echo '<strong>Goal Name: ' . $goal->{'name'} . '</strong><br />' . PHP_EOL;
-						echo ' (rate: ' . $goal->{'payout_rate'} . ' ' . $campaign->{'currency'} . ' payout type: ' . $goal->{'payout_type'} . ' max_per_visitor: ' . $goal->{'max_per_visitor'} . ' max_per_promoter: ' . $goal->{'max_per_promoter'} . ' payouts: ' .  $goal->{'payouts'} . ')<br />' . PHP_EOL;
-						echo 'Funding address: ' . $goal->{'funding_Address'} . ' ' . PHP_EOL;
+						echo '<strong>' . esc_html( __( 'Goal Name:', TONICPOW ) ) . ' ' . $goal->{'name'} . '</strong><br />' . PHP_EOL;
+						echo ' (' . esc_html( __( 'rate:', TONICPOW ) ) . ' ' . $goal->{'payout_rate'} . ' ' . $campaign->{'currency'} . ' ' . esc_html( __( 'payout type:', TONICPOW ) ) . ' ' . $goal->{'payout_type'} . ' ' . esc_html( __( 'max_per_visitor:', TONICPOW ) ) . ' ' . $goal->{'max_per_visitor'} . ' ' . esc_html( __( 'max_per_promoter:', TONICPOW ) ) . ' ' . $goal->{'max_per_promoter'} . ' ' . esc_html( __( 'payouts:', TONICPOW ) ) . ' ' . $goal->{'payouts'} . ')<br />' . PHP_EOL;
+						if ( ! empty( $goal->{'funding_Address'} ) ) {
+							echo '' . esc_html( __( 'Funding address:', TONICPOW ) ) . ' ' . $goal->{'funding_Address'} . ' ' . PHP_EOL;
+						}
 					}
 					// todo: fix the url below to be dynamic
-					echo '<a target="_blank" href="https://tonicpow.com/app/dashboard/ad-profiles/' . $campaign->{'advertiser_profile_id'} . '/campaigns/' . $campaign->{'id'} . '/goals">Edit</a>' . PHP_EOL;
+					echo '<a target="_blank" href="https://tonicpow.com/app/dashboard/ad-profiles/' . $campaign->{'advertiser_profile_id'} . '/campaigns/' . $campaign->{'id'} . '/goals">' . esc_html( __( 'Edit', TONICPOW ) ) . '</a>' . PHP_EOL;
 					echo '</li>' . PHP_EOL;
 				} else {
-					echo '<li style="border: 1px solid indianred; border-radius: 6px; margin-bottom: .25rem; padding: .25rem;"><strong>' . $campaign->{'title'} . '</strong> has no conversion goals. <a target="_blank" href="https://web.staging.tonicpow.com/app/dashboard/ad-profiles/' . $campaign->{'advertiser_profile_id'} . '/campaigns/' . $campaign->{'id'} . '/goals">Set one up</a></li>' . PHP_EOL;
+					echo '<li style="border: 1px solid indianred; border-radius: 6px; margin-bottom: .25rem; padding: .25rem;"><strong>' . $campaign->{'title'} . '</strong> ' . esc_html( __( 'has no conversion goals.', TONICPOW ) ) . ' <a target="_blank" href="https://web.staging.tonicpow.com/app/dashboard/ad-profiles/' . $campaign->{'advertiser_profile_id'} . '/campaigns/' . $campaign->{'id'} . '/goals">' . esc_html( __( 'Set one up', TONICPOW ) ) . '</a></li>' . PHP_EOL;
```

Comment:
Hardcoded staging url - needs to be replaced with the production url

`https://web.staging.tonicpow.com` for `https://tonicpow.com`